news/12604-fix-urlsdata-get-url

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* `conda.core.package_cache_data.UrlsData.get_url` no longer fails when `package_path` has `.conda` extension (#12536)
```

Comment:
```suggestion
* `conda.core.package_cache_data.UrlsData.get_url` no longer fails when `package_path` has `.conda` extension (#12516 via #12604)
```

Reply:
Ah whoops my bad was looking at another example of how to do the news file and accidently used their pr/issue number instead

Reply:
No probs, that's what code reviews are for! You have done a great job with the tests and everything! 💪